src/java/org/jivesoftware/openfire/crowd/CrowdAuthProvider.java

```diff
@@ -47,9 +47,6 @@ public CrowdAuthProvider() {
      * Returns if the username and password are valid; otherwise this
      * method throws an UnauthorizedException.<p>
      *
-     * If {@link #isPlainSupported()} returns false, this method should
-     * throw an UnsupportedOperationException.
-     *
```

Comment:
Was this removed because a technical javadoc issue (that I can't see), or because the statement itself is incorrect?